Stockfish/SFMWindowController.m

```diff
@@ -9,12 +9,16 @@
 #import "SFMWindowController.h"
 #import "SFMChessGame.h"
 #import "Constants.h"
+#import "SFMArrowMove.h"
 #import "SFMFormatter.h"
 #import "SFMMove.h"
 #import "SFMUCILine.h"
 #import "SFMPosition.h"
 #import "SFMUserDefaults.h"
 
+const CGFloat kMinWeight = 0.25;
```

Comment:
Use a min weight higher than 0, otherwise, the weakest variation's arrow width will be mutliplied by 0, making it invisible.